books/free-programming-books-langs.md

```diff
@@ -1389,8 +1389,11 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### Jenkins
 
+* [Jenkins Essentials](https://raw.githubusercontent.com/rootusercop/Free-DevOps-Books-1/6dd8ee5bef93d2a1489a3761515ecbaf16c118cc/book/Jenkins%20Essentials.pdf) (PDF)
```

Comment:
This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines. Remove it